scripts/test/ingestBatcher/system-test-instance.properties

```diff
@@ -252,35 +96,5 @@ sleeper.systemtest.writers=110
 # The number of random records that each container should write
 sleeper.systemtest.records-per-writer=40000000
```

Comment:
These three properties aren't used for the ingest batcher. It's actually not deployed as a system test instance, so this section isn't relevant.